.github/workflows/build-images.yml

```diff
@@ -223,8 +223,13 @@ jobs:
         run: |
           rm -rf "scripts/ci"
           mv "main-airflow/scripts/ci" "scripts"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - run: python -m pip install --editable ../../dev/breeze/
```

Comment:
No need for the ../../ -> iot should really be `./dev/breeze` because this is where our sub-project resides